src/Essential.Diagnostics.RollingFileTraceListener/Diagnostics/RollingFileTraceListener.cs

```diff
@@ -111,6 +112,36 @@ public bool ConvertWriteToEvent
             }
         }
 
+        private bool? _newStreamOnError = null;
```

Comment:
Probably does not need a backing field, just store in the attributes. Also, no need for nullable - either you create a new stream or you don't, so it only needs true/false.